Add deployed version link and fix typo (#16383)
MystenLabs/sui: docs/content/guides/developer/app-examples/blackjack.mdx

The following documentation goes through an example implementation of the popular casino game blackjack on Sui. This guide walks through its components, providing a detailed look at the module's functions, structures, constants, and their significance in the overall gameplay mechanism.

> A deployed version of the blackjack game can be found [here](https://blackjack-sui.vercel.app/).

Reviewer: ```suggestion
A deployed version of the blackjack game is online at [Mysten Blackjack](https://blackjack-sui.vercel.app/).
```
